src/core_ocean/shared/mpas_ocn_time_varying_forcing.F

```diff
@@ -0,0 +1,573 @@
+!         atmosBioFluxes, &
+!         atmosBlackCarbonFlux, &
+!         atmosDustFlux
+!
+!    logical, pointer :: &
+!         config_use_column_biogeochemistry
+!
+!    block => domain % blocklist
+!    do while (associated(block))
+!
+!       ! physics
+!       call MPAS_pool_get_subpool(block % structs, "atmos_fluxes", atmosFluxes)
+!       call MPAS_pool_get_subpool(block % structs, "shortwave", shortwave)
+!       call MPAS_pool_get_subpool(block % structs, "velocity_solver", velocitySolver)
+!       call MPAS_pool_get_subpool(block % structs, "atmos_coupling", forcingPool)
+!
+!       call MPAS_pool_get_array(atmosFluxes, "sensibleHeatFlux", sensibleHeatFlux)
+!       call MPAS_pool_get_array(atmosFluxes, "latentHeatFlux", latentHeatFlux)
+!       call MPAS_pool_get_array(atmosFluxes, "longwaveUp", longwaveUp)
+!       call MPAS_pool_get_array(atmosFluxes, "evaporativeWaterFlux", evaporativeWaterFlux)
+!
+!       call MPAS_pool_get_array(velocitySolver, "airStressCellU", airStressCellU)
+!       call MPAS_pool_get_array(velocitySolver, "airStressCellV", airStressCellV)
+!
+!       call MPAS_pool_get_array(shortwave, "absorbedShortwaveFlux", absorbedShortwaveFlux)
+!
+!       call MPAS_pool_get_array(forcingPool, 'atmosReferenceSpeed10m', atmosReferenceSpeed10m)
+!       call MPAS_pool_get_array(forcingPool, 'atmosReferenceTemperature2m', atmosReferenceTemperature2m)
+!       call MPAS_pool_get_array(forcingPool, 'atmosReferenceHumidity2m', atmosReferenceHumidity2m)
+!
+!       sensibleHeatFlux            = 0.0_RKIND
+!       latentHeatFlux              = 0.0_RKIND
+!       longwaveUp                  = 0.0_RKIND
+!       evaporativeWaterFlux        = 0.0_RKIND
+!
+!       airStressCellU              = 0.0_RKIND
+!       airStressCellV              = 0.0_RKIND
+!
+!       absorbedShortwaveFlux       = 0.0_RKIND
+!
+!       atmosReferenceSpeed10m      = 0.0_RKIND
+!       atmosReferenceTemperature2m = 0.0_RKIND
+!       atmosReferenceHumidity2m    = 0.0_RKIND
+!
+!
+!       block => block % next
+!    end do
+!
+!  end subroutine reset_atmospheric_coupler_fluxes
+
+!-----------------------------------------------------------------------
+! restart
+!-----------------------------------------------------------------------
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  ocn_time_varying_forcing_write_restart_times
+!
+!> \brief
+!> \author Steven Brus, LANL
+!> \date
+!> \details
+!>
+!
+!-----------------------------------------------------------------------
+
+  subroutine ocn_time_varying_forcing_write_restart_times(domain)
+
+    type(domain_type) :: domain
+
+    logical, pointer :: &
+         config_use_atmospheric_forcing
+
+    call MPAS_pool_get_config(domain % configs, "config_use_time_varying_atmospheric_forcing", config_use_atmospheric_forcing)
+
+    if (config_use_atmospheric_forcing) then
+
+       call MPAS_forcing_write_restart_times(forcingGroupHead)
```

Comment:
@sbrus89, was @akturner able to give you any clarity on this call?

Reply:
@xylar, he was. He said it should be only called once for the whole forcingGroupHead. I tried to address that here:

https://github.com/MPAS-Dev/MPAS-Model/blob/2b37800b013202204437290bed3e567b7e159cab/src/core_ocean/mode_forward/mpas_ocn_forward_mode.F#L638-L645

If you (or anyone else) can think of a better way of doing this, I'm all ears...